sox/transform.py

```diff
@@ -1646,8 +1707,11 @@ def echo(self, gain_in=0.8, gain_out=0.9, n_echos=1, delays=[60],
         self.effects_log.append('echo')
         return self
 
-    def echos(self, gain_in=0.8, gain_out=0.9, n_echos=1, delays=[60],
-              decays=[0.4]):
+    def echos(self,
+              gain_in: float = 0.8, gain_out: float = 0.9,
+              n_echos: int = 1,
+              delays: Sequence[float] = (60,),
+              decays: Sequence[float] = (0.4,)):
```

Comment:
I'd use `List[float]` here

Reply:
I only say this for consistency, since there are lots of other places where variables could be generalized Sequences and we use Lists. Happy either way though!

Reply:
Yeah, regarding those, it's because the default args: Pycharm's static checker is making me feel bad for having lists in the default arguments. i'll switch back to lists.